src/test/resources/after.pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.pekko</groupId>
+    <artifactId>pekko-stream_3</artifactId>
+    <packaging>jar</packaging>
+    <description>Apache Pekko is a toolkit for building highly concurrent, distributed, and resilient message-driven applications for Java and Scala.</description>
+    <url>https://pekko.apache.org/</url>
+    <version>1.0.2</version>
+    <licenses>
+        <license>
+            <name>Apache-2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <name>apache-pekko-stream</name>
```

Comment:
it's probably not absolutely necessary but is there is any chance that this could be modified so that it is title cased without the `-` chars as `Apache Pekko Stream`?

Reply:
Its definitely possible, there just isn't clarity on whether it should be the human readable form or just the `apache-` prefix

Reply:
So it would be better a  `String => String` function

Reply:
@pjfanning Updated the PR to put in a human readable name, let me know if its good and then I can merge + publish